source/_integrations/ezviz.markdown

```diff
@@ -17,231 +19,11 @@ It also exposes an RTSP stream, by using the local camera IPs (so the device hos
 
 As there is no official documentation for the API, the component retrieves data from the API used in the Ezviz mobile app, [hosted here](https://apiieu.ezvizlife.com).
 
-## Configuration
-
-The configuration (see below) needs your Ezviz username & password you use within the Ezviz mobile application.
-
-Next, add the Ezviz camera to your `configuration.yaml` file like below:
-
-```yaml
-# Example configuration.yaml entry
-camera:
-  - platform: ezviz
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-    cameras:
-      D12345678:
-        username: YOUR_CAMERA_USERNAME
-        password: YOUR_CAMERA_PASSWORD
-```
-
 As you see, here is the way to describe the camera credentials.
 The password for each camera is usually written underneath the device, near the QR code. It is also code "verification code".
 
-{% configuration %}
-username:
-  description: The Ezviz account username.
-  required: true
-  type: string
-password:
-  description: The Ezviz account password.
-  required: true
-  type: string
-cameras:
-  description: A list of cameras
-  required: false
-  type: string
-  serial:
-    description: The Ezviz camera Serial, usually a letter followed by 8 digits.
-    required: true
-    type: string
-    username:
-      description: The Ezviz camera RTSP username.
-      required: true
-      type: string
-    password:
-      description: The Ezviz verification code.
-      required: true
-      type: string
-{% endconfiguration %}
-
-## Lovelace
-
-If you want a shiny Lovelace card, with proper buttons, here is a snippet of a Lovelace card:
+{% include integrations/config_flow.md %}
 
-```yaml
+Your cameras will now show under integration options as "discovered devices". Please complete the setup for each camera to see the video stream within Home Assistant.
```

Comment:
Devices discovery is already explained in the included config flow snippet, is this an additional step?

Reply:
Hi @frenck,

The integration discovers cameras linked to the ezviz cloud account. The user then needs to "complete the setup" for the discovered cameras by providing a username and password.